### ISSUE TITLE
No button to create wiki

### ISSUE THREAD

**Issue by dantman** [Jermolene/TiddlyDesktop#55]
Trying out 0.0.4 on OSX for the first time. There doesn't appear to be any option to create a brand new wiki. Only a button to open an existing tiddilywiki.

**Comment by Spangenhelm** [comment]
Hi, i think it is not yet available (but it seems planned though, check the release log)
It would be a practical functionality..

+++ Spangenhelm +++

**Comment by Spangenhelm** [comment]
Shall we close this issue ? Thank you

**Comment by dantman** [comment]
Why? As I understand the functionality still isn't available. It doesn't really make sense to close the bug until that is fixed.

**Comment by Spangenhelm** [comment]
Well it is up to you but since this is more of a lack of functionnality than a bug and that it is already planned in the roadmap, i don't see the point of leaving it open if there is nothing more we can do to help.

+

**Comment by dantman** [comment]
I consider this more of a tracker than a support forum. Letting others with this issue subscribe to this bug and closing it once the create wiki button exists so others know they can use it makes more sense to me.

**Comment by Spangenhelm** [comment]
ok

**Comment by Jermolene** [comment]
Hi @dantman the plan is indeed to provide a toolbar button to create new wiki files from a menu of empty editions.

**Comment by danielo515** [comment]
I have to agree with this suggestion. Since TiddlyDesktop it's a wiki itself, and it includes all necessary "bits" to create brand new wikis this is an important usability lack. Please add my vote to this.

**Comment by joepie91** [comment]
What is the suggested method to create a new wiki for TiddlyDesktop right now?
